### DISCUSSION TITLE
Set Popover transition effect according to current position

### DISCUSSION THREAD

**Discussion by u3u** [mantinedev/mantine#1204] [Feature requests]
I want the transition to be bottom-up if the current `position` is `top` and top-down if the `position` is `bottom`
There doesn't seem to be a way to implement this at the moment, can we consider passing `transition` as a function?

`transition: MantineTransition | (params: { position: string, placement: string }) => MantineTransition`

https://github.com/mantinedev/mantine/blob/b5d341b1978e35c759ea1b1d6ba7b18a39668683/src/mantine-core/src/components/Popper/Popper.tsx#L155-L169

**Comment by rtivital** [comment]
No, currently, there is no way of subscribing to position changes. We may work on this feature in the next major release.
